cert_manager/Tiltfile

```diff
@@ -26,27 +26,26 @@ spec:
 """
 
 # Deploys cert manager to your environment
-def deploy_cert_manager(registry="quay.io/jetstack", version="v0.16.1", load_to_kind=False, kind_cluster_name="kind"):
+def deploy_cert_manager(registry="quay.io/jetstack", version="v1.1.0", load_to_kind=False, kind_cluster_name="kind"):
     silent=True
+    if version.startswith('v0'):
+      cert_manager_test_resources_versioned = cert_manager_test_resources.format(cert_manager_api_version='v1alpha2')
+    else:
+      cert_manager_test_resources_versioned = cert_manager_test_resources.format(cert_manager_api_version='v1')
 
-    # check if cert-mamager is already installed, otherwise pre-load images & apply the manifest
-    # NB. this is required until https://github.com/jetstack/cert-manager/issues/3121 is addressed otherwise
-    # when applying the manifest twice to same cluster kubectl get stuck
-    existsCheck = str(local("kubectl get namespaces", quiet=silent, echo_off=silent))
-    if existsCheck.find("cert-manager") == -1:
```

Comment:
I'm not sure we can drop this hack for cert manager < v1.0.0
You can test this by doing tilt up twice with cert-manager v0.16.1 (or by applying the cert-manager manifest for v1.16.1 twice on the same cluster).

Reply:
It seems working. As I understand looking at the issue it has been fixed directly on Kubernetes and Helm.
Probably this can create some issues for users with older versions of k8s. I do not know if we want that.

Reply:
Ok then for me.
It would be nice to add a comment near `kubectl apply -f` to keep track that this could not work when executing "tilt up" more times on the same cluster with older version of kubernetes/certmanager (and link to the issue)

Reply:
Let me know if it is fine.